main/src/ca/uwaterloo/flix/language/fmt/SimpleType.scala

```diff
@@ -440,7 +440,19 @@ object SimpleType {
           }
 
         case TypeConstructor.Extensible =>
-          ??? // TODO: Ext-Variants
+          val args = t.typeArguments.map(visit)
+          args match {
+            // Case 1: No args. { ? }
+            case Nil => SchemaConstructor(Hole)
```

Comment:
i still think its a mess to confuse the schema constructor with the extensible variant constructor

Reply:
We need to collapse them